services/markets/coinmarketcap/mapping.go

```diff
@@ -13726,6 +13726,12 @@ const Mapping = `[
         "token_id": "0x2DBd330bC9B7f3A822a9173aB52172BdDDcAcE2A",
         "id": 7358
     },
+    {
+        "coin": 20000714,
```

Comment:
Should coin be 60? This is ethereum

Reply:
This is actually for BEP20, hence the 20000714
https://bscscan.com/token/0xE40255C5d7fa7ceEc5120408C78C787CECB4cfdb